code/modules/admin/IsBanned.dm

```diff
@@ -6,7 +6,7 @@
 #define STICKYBAN_MAX_EXISTING_USER_MATCHES 5 //ie, users who were connected before the ban triggered
 #define STICKYBAN_MAX_ADMIN_MATCHES 2
 
-/world/IsBanned(key,address,computer_id)
+/world/IsBanned(key,address,computer_id,real_bans_only=FALSE)
 	if (!key || !address || !computer_id)

```

Comment:
add the check here too. goon chat may call it with null addresses and cids and the other code already knows how to handle that

Reply:
Done